index.js

```diff
@@ -1,18 +1,79 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
-}
+const readline = require("readline"); {
+  const rl = readline.createInterface(process.stdin, process.stdout);
+
+  function ask(questionText) {
+    return new Promise((resolve, reject) => {
+      rl.question(questionText, resolve);
+    });
+  }
+
+  function randomNum(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  }
+  start();
+  async function start() {
+    console.log(
+      "Let's play a game where you (Joe) make up a number and I (computer) try to guess it."
+    );
+
+    let secretNum = await ask(
+      "What is your secret number?\nI won't peek, I promise...\n"
+    );
+    let min = 0; //min
+    let max = 100; //max
+    let contestantOne = randomNum(min, max); //computers guess
```

Comment:
This would be the only variable I would say could be better. Having it called contestantOne could be talking about a bunch of different things. You have it commented saying that it is the computer guess. Calling it compGuess or something similar would be a more descriptive variable name